source/vistas/ui/windows/linreg_dialog.py

```diff
@@ -11,54 +11,374 @@
+        self.mouse_y = 0
+
+        # mouse drag distance
+        self.mouse_x_diff = 0
+        self.mouse_y_diff = 0
+
+        # Current bounds of the graph
+        self.x_hi = 0
+        self.x_lo = 0
+        self.y_hi = 0
+        self.y_lo = 0
+
+        # Center of the viewport
+        self.center_x = 0
+        self.center_y = 0
+
+        #SIZERS
         self.panel = wx.Panel(self)
-        self.sizer = wx.BoxSizer(wx.VERTICAL)
+        self.sizer = wx.BoxSizer(wx.VERTICAL) #Main sizer
         self.panel.SetSizer(self.sizer)
-        top_sizer = wx.BoxSizer(wx.HORIZONTAL)
+        top_sizer = wx.BoxSizer(wx.HORIZONTAL) #Sizer to divide screen into left and right halves
         self.sizer.Add(top_sizer, 0)
-        ctl_sizer = wx.BoxSizer(wx.VERTICAL)
+        ctl_sizer = wx.BoxSizer(wx.VERTICAL) #Sizer for the controls on left size of the screen
         top_sizer.Add(ctl_sizer, 0)
+        zoom_sizer = wx.BoxSizer(wx.HORIZONTAL) #Sizer for the zoom controls
+        self.right_sizer = wx.BoxSizer(wx.VERTICAL) #Sizer for the right half of the window
+
+        #Variables title
         ctl_sizer.Add(wx.StaticText(self.panel, -1, 'Variables:'), flag=wx.TOP|wx.LEFT|wx.RIGHT, border=20)
 
+        #Get choices for independent and dependent variables
         self.data = Project.get().all_data
         data_choices = [n.data.data_name for n in self.data]
 
+        #INDEPENDENT VARIABLE
         ctl_sizer.Add(wx.StaticText(self.panel, -1, 'Independent variable(s):'), flag=wx.TOP|wx.LEFT|wx.RIGHT, border=12)
+
         self.iv_chooser = wx.ListBox(self.panel, choices=data_choices, style=wx.LB_EXTENDED)
         ctl_sizer.Add(self.iv_chooser, flag=wx.LEFT|wx.RIGHT, border=12)
 
-        ctl_sizer.Add(wx.StaticText(self.panel, -1, 'Dependent variable'), flag=wx.TOP|wx.LEFT|wx.RIGHT, border=12)
+        #DEPENDENT VARIABLE
+        ctl_sizer.Add(wx.StaticText(self.panel, -1, 'Dependent variable:'), flag=wx.TOP|wx.LEFT|wx.RIGHT, border=12)
+
         self.dv_chooser = wx.Choice(self.panel, choices=['-'] + data_choices)
         ctl_sizer.Add(self.dv_chooser, flag=wx.LEFT|wx.RIGHT, border=12)
 
+        #PLOT TYPE
         ctl_sizer.Add(wx.StaticText(self.panel, -1, 'Plot type:'), flag=wx.TOP|wx.LEFT|wx.RIGHT, border=12)
+
         self.plot_type = wx.RadioBox(self.panel, choices=['scatterplot', 'heatmap'])
         ctl_sizer.Add(self.plot_type, flag=wx.LEFT|wx.RIGHT, border=10)
+
+        #AXIS TYPE
         ctl_sizer.Add(wx.StaticText(self.panel, -1, 'Axis type:'), flag=wx.TOP|wx.LEFT|wx.RIGHT, border=12)
-        self.axis_type = wx.RadioBox(self.panel, choices=['fixed', 'adaptive'])
+
+        self.axis_type = wx.RadioBox(self.panel, choices=['Fit All', 'Adaptive', 'Zoom'])
         ctl_sizer.Add(self.axis_type, flag=wx.LEFT|wx.RIGHT, border=10)
+
         self.Bind(wx.EVT_RADIOBOX, self.doPlot)
 
+        #BLANK SPACER
+        ctl_sizer.Add(wx.StaticText(self.panel, -1, ''), flag=wx.TOP|wx.EXPAND, border=200)
```

Comment:
It would be better to use `.AddSpacer()` here.

https://wxpython.org/Phoenix/docs/html/wx.Sizer.html#wx.Sizer.AddSpacer